index.html

```diff
@@ -4,28 +4,32 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Jeff Gebhardt</title>
+    <title>Jeff Gebhardt - Home</title>
 
     <!-- CSS -->
     <link rel="stylesheet" type="text/css" media="screen" href="css/style.css" />
     <link rel="stylesheet" type="text/css" media="screen and (min-width:480px)" href="css/mediumScreen.css" />
     <link rel="stylesheet" type="text/css" media="screen and (min-width:980px)" href="css/largeScreen.css" />
-    
+
 </head>
 
 <body>
     <div class="container">
```

Comment:
Do you really need this div, or the mainContainer div? It's not bad, just a question of how much you really need to make your page.